src/cmdlib.sh

```diff
@@ -126,10 +142,86 @@ EOF
     fi
 
     rm -f ${changed_stamp}
-    set -x
-    sudo rpm-ostree compose tree --repo=${workdir}/repo-build --cachedir=${workdir}/cache \
-         --touch-if-changed "${changed_stamp}" \
-         ${treecompose_args} \
-         ${TREECOMPOSE_FLAGS:-} ${manifest} "$@"
-    set +x
+
+    set - rpm-ostree compose tree --repo=${workdir}/repo \
+            --cachedir=${workdir}/cache --touch-if-changed "${changed_stamp}" \
+            ${manifest} "$@"
+
+    if ! grep -q '^# disable-unified-core' "${manifest}"; then
+        set - "$@" --unified-core
```

Comment:
colin had told me that I could drop this check so I did [in my fork](https://github.com/dustymabe/coreos-assembler/commit/965a0772a5f82f7fbfb204dedb34c8ee8246230c). i.e. it is safe to assume anything using coreos-assembler will be using unified core in the future.